src/panel/pages/request/Request.tsx

```diff
@@ -1,27 +1,82 @@
 import React, { ComponentProps, FC } from "react";
 import styled from "styled-components";
 import { Background } from "../../components/Background";
-import { Query, Response, Settings } from "./components";
+import { Pane } from "../../components";
+import { Query, Schema, Settings, Response } from "./components";
 
 export const Request: FC<ComponentProps<typeof Page>> = (props) => {
   return (
     <Page {...props}>
       <Settings />
       <PageContent>
         <Query />
-        <Response />
+        {/* {TODO: Hack to offset the panels so they aren't on top of each other.
+       There's definitely better way to do this */}
+        <Pane initSize={{ y: 700, x: 400 }}>
+          <PaneBody>
+            <PaneSection>
+              <Response />
+            </PaneSection>
+          </PaneBody>
+          {/* {TODO: Also a hack to make these panes work together, sorry! */}
```

Comment:
No worries about the hack, the Pane component was the one that needed the most work, what you've got is great. For this PR I'm more than happy with this. We can always build out a more specced out API for managing nested Panes later on.